submission_rules.adoc

```diff
@@ -267,6 +271,22 @@ A submission must contain the following:
 *   Result logs for each system-implementation tested
 
 
+The submission must be submitted by the submission deadline either as a pull request to the submission repository or using the <<id_submission_privacy,private submission process>>. 
+
+#### Qualified Exceptions [[id_qualified_exceptions]]
+
+In the case where last-minute unforeseen events prevent a submission from being submitted to the submission repository or sent to the results chair, a Late Submitter is allowed to make a submission after the deadline following the procedure outlined below. This procedure is meant for extreme circumstances beyond the submitter's control. No submitter may make a Late Submission in two consecutive rounds unless by unanimous approval from the other submitters.
```

Comment:
Why is a late submission for two consecutive rounds prohibited (other than with unanimous approval)? This seems arbitrary and vague. If there is a specific purpose for this limitation, then the purpose should be included as part of the rule.

Reply:
This is to prevent this option from getting abused. We anticipate that a submitter who has had to use this option in one round will be much better prepared in the next round (i.e. by submitting early/often using the private submission process).

Reply:
Okay, that makes sense. But "unanimous approval" is perhaps too strong for this specific case. I find it difficult to accept such a strict sentiment in regards to a speculative situation that currently is not typically encountered nor intentionally abused (yet). Again, the "feel" of this change is that the rules get more complicated over no apparent problem.